src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -68,6 +72,12 @@ public void sendPaymentConfirmedEmailFor(Transaction transaction, Payer payer, L
         return map;
     }
 
+    private HashMap<String, String> buildPaymentProblemPersonalisation() {
```

Comment:
A better name for this method will make it much clearer. It looks like is putting DD Guarantee into a map, quite generic might be as simple as `buildPersonalisation()`